docs/cheaha/migration_overview.md

```diff
@@ -0,0 +1,194 @@
+    1. Ceph only stores data deemed to be inactive on Cheaha.
+1. **CephFS is NOT something users will interact with**
+    1. Users will never make direct contact with Ceph. All of this is purely informational and provides background to some changes and behavior explained below.
+    1. All data transfer to and from Ceph is done transparently, without any direct action from the user. Do not change how you interact with Cheaha on account of this tiered storage.
+
+<!-- markdownlint-disable MD046 -->
+!!! critical
+    It is imperative to understand that tiered storage does not equal a backup. We do not provide a traditional, automatic backup for data stored on Cheaha. All data are erasure-encoded in case of hardware failure, they are not backed up in case of user error. Please see information about [LTS](../data_management/lts/index.md) for a potential backup solution.
+<!-- markdownlint-enable MD046 -->
+
+### What CephFS IS
+
+1. Cost-efficient, expandable storage on Cheaha.
+    1. Increasing GPFS storage capacity is cost-prohibitive and has severely limited our ability to grant increases to project space quotas.
+    1. CephFS provides a **FUTURE** avenue for expandable storage. **This is not currently an option**. More details will be provided when appropriate.
+1. Tunable data sink
+    1. With Ceph, we are able to tune which files are offloaded from GPFS based on user behavior and activity without interfering with ongoing analysis or burdening the researcher with quickly moving data to another storage system.
+    1. This allows us a powerful option to address potential GPFS storage constraints that has not been available previously.
+
+### Tiered Storage Summary
+
+In summary, **users should not take tiered storage into account when using Cheaha**.
+
+- There will be no changes in user experience when traversing a directory tree as file stubs will appear as normal files from the user's perspective.
+- Slight delays when fetching old data should be expected as the data is migrated from Ceph to GPFS. See [below](#initial-interaction-with-files)
+- Tiered storage is NOT back-up storage
+- Tiered storage IS a new tool to improve storage needs on Cheaha.
+- See notes on [quotas](#quotas) below
+
+## GPFS 5 Notes Post-Migration
+
+### Initial Interaction With Files
+
+After migration to GPFS 5, loading a given file will be slow during the initial read but will return to normal for all subsequent reads. This is an expected behavior of our new tiered file system compared to our GPFS 4 system. This will be especially evident during startup of interactive apps from the web portal and during activation and use of virtual environments such as `conda` due to the large number of files these tools use. Please be patient when starting an app such as Jupyter Notebook or reading an especially large file for the first time post-migration due to the delay caused by initial file reads.
+
+### Jobs Requeued Prior to Migration
```

Comment:
Do we have advice for this? Like, "consider manually ending your jobs before we initiate migration to avoid ..."

Reply:
I'm confused about how this comment relates to this snippet. Is it supposed to go with the Jobs Requeued Prior to Migration section below it?

Reply:
Yes, I think I misclicked, it was meant for the next section.